calblueprint/alemany
# added jest testing setup for expo

## Overview
- Enabled jest tests for unit and snapshot testing.
- Added example test for EditScreenInfo.tsx
- Configured setup to play well with prettier, eslint, and babel configs
- Toggled code coverage for tests 

TODO: look into screenshot testing and accessibility selectors

## Helpful Sources
### Setup
- https://keyholesoftware.com/2020/11/02/using-jest-and-testing-library-with-react-native-part-1-setting-up/
- https://docs.expo.dev/guides/testing-with-jest/

### Using Tests
- https://callstack.github.io/react-native-testing-library/docs/getting-started
- https://jestjs.io/docs/snapshot-testing

## Testing
- Pull this branch.
- `yarn test`
- Tests should pass with the below results.
- Navigate to `coverage/icov_report/index.html`



![image](https://user-images.githubusercontent.com/35586281/134629678-e8beac3a-e0ae-4c03-bd08-3b2630734637.png)
